ui/js/component/fileActions/index.js

```diff
@@ -48,6 +53,9 @@ const perform = dispatch => ({
   startDownload: uri => dispatch(doPurchaseUri(uri, "affirmPurchase")),
   loadVideo: uri => dispatch(doLoadVideo(uri)),
   restartDownload: (uri, outpoint) => dispatch(doStartDownload(uri, outpoint)),
+  sendToAddress: () => dispatch(doSendDraftTransaction()),
+  setAmount: amount => dispatch(doSetDraftTransactionAmount(amount)),
```

Comment:
I'm unsure. This might need some testing/confirmation from your side. Although to make an educated guess, we _are_ dispatching states in `doSendDraftTransactions`, so it must be to avoid these weird behaviour. But I am not sure if that's how it works.